Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -469,6 +587,30 @@ Node::InsertedIntoAncestorResult HTMLAttachmentElement::insertedIntoAncestor(Ins
         setInlineStyleProperty(CSSPropertyMarginTop, 1, CSSUnitType::CSS_PX);
         setInlineStyleProperty(CSSPropertyMarginBottom, 1, CSSUnitType::CSS_PX);
     }
+
+    if (m_docInsertionLoggedCount < docInsertionLoggedMax) {
+        RELEASE_LOG(TextManipulation, "HTMLAttachmentElement[%p]::insertedIntoAncestor(type.connectedToDocument=%d .treeScopeChanged=%d)", this, int(type.connectedToDocument), int(type.treeScopeChanged));
```

Comment:
`TextManipulation` (a.k.a. Safari webpage translation) seems like the wrong logging category to use. Adding a new `Attachments` category seems fine, or we could just use `Editing`.